src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,51 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Cars {
+    private List<Car> cars;
+
+    public Cars(int carNumber) {
+        cars = new ArrayList<>();
+        initializeCars(carNumber);
+    }
+
+    public void initializeCars(int carNumber) {
+        for (int i = 0; i < carNumber; i++) {
+            this.addCar(new Car());
+        }
+    }
```

Comment:
생성을 위한 메서드에서 호출되어야 하니 
```suggestion
    public Cars(int carNumber) {
        cars = new ArrayList<>();
        initializeCars(carNumber);
    }

    private void initializeCars(int carNumber) {
        for (int i = 0; i < carNumber; i++) {
            this.addCar(new Car());
        }
    }
```
외부에서 접근할 수 없도록 private 해야하지 않을까요?

Reply:
carNumber로 0 이하의 값이 들어온다면 적절한 예외가 필요할 것 같아요
이에 대한 테스트 코드도 있다면 좋겠습니다.

Reply:
피드백 감사합니다.